attest/core/src/quote.rs

```diff
@@ -333,6 +332,61 @@ impl Quote {
 
         Ok(())
     }
+
+    fn verify_report_body(
```

Comment:
This logic is pretty much copied from, https://github.com/mobilecoinfoundation/mobilecoin/blob/36be4bf1773ecb302c2204f09a3a15563da21333/attest/core/src/types/report_body.rs#L144